cron/worker/main.go

```diff
@@ -84,11 +72,15 @@ func processRequest(ctx context.Context,
 	filename := data.GetBlobFilename(
 		fmt.Sprintf("shard-%05d", batchRequest.GetShardNum()),
 		batchRequest.GetJobTime().AsTime())
-	return errors.Unwrap(data.WriteToBlobStore(ctx, bucketURL, filename, buffer.Bytes()))
+	if err := data.WriteToBlobStore(ctx, bucketURL, filename, buffer.Bytes()); err != nil {
+		return fmt.Errorf("error during WriteToBlobStore: %w", err)
+	}
+	log.Printf("Write to shard file successful: %s", filename)
```

Comment:
is this a debug message, will this show in regular runs?

Reply:
Yes, it will show in regular run. Useful to find the file dumped out by the worker.

Reply:
I meant does it show when running the scorecard tool on command line ?

Reply:
No, only run during the cron job.